packages/grid/test/visual/lumo/grid.test.js

```diff
@@ -255,14 +257,6 @@ describe('grid', () => {
         });
 
         it('row focus - header', async () => {
-          // Add a header group
-          const group = document.createElement('vaadin-grid-column-group');
-          group.header = 'Address';
-          group.appendChild(element.querySelector('[path="location.city"]'));
-          element.appendChild(group);
-          flushGrid(element);
-          await nextRender(element);
-
```

Comment:
Cleaned up this test to not add the column group dynamically, (which was actually buggy with the previous version; wrong header row was made focusable).

Added a static column group so needed to update the screenshots for both of the row focus tests.